public/v3dApp/visual_logic.js

```diff
@@ -14813,6 +14854,7 @@ function createPL(v3d = window.v3d) {
       }
       _basic_module_id = dictGet(_STATE_basic_module, "id");
       if (dictGet(_STATE_skeleton, "cam_addon_target")) {
+        console.log("case 1 : ");
```

Comment:
I guess it's left over, and there are more

Reply:
of course, I need second erros information